package.json

```diff
@@ -80,6 +80,7 @@
     "core-js": "3.6.5",
```

Comment:
we use yarn instead of npm. We already have a yarn.lock. please use yarn to install dependencies